### ISSUE TITLE
No effects after installing

### ISSUE THREAD

**Issue by amivaleo** [mzur/gnome-shell-always-indicator#7]
Hi,

I can't really notice any difference after installing this extension on my computer (Fedora 31, gnome 3.34.4, wayland session): the blue dot doesn't appear, ever.

I installed this extension from the website: https://extensions.gnome.org/extension/2594/always-indicator/

I usually have notifications from gnome-calendar, geary, spotify. The dot never shows up.

A try I did: I gave the command

`notify-send "something"`

from a terminal. The dot doesn't show up.

When should the dot appear? What am I doing wrong?

**Comment by mzur** [comment]
Can you please take a look at `journalctl /usr/bin/gnome-shell` and search for any error message that might belong to this extension?

**Comment by amivaleo** [comment]
Not sure how to do that.


I disabled all the extensions I use except for this one, rebooted my laptop and then I gave

```
$ journalctl /usr/bin/gnome-shell | grep "always-indicator"
feb 18 08:39:42 jimmy-xps-9360 gnome-shell[1710]:   inflating: /home/jimmy/.local/share/gnome-shell/extensions/always-indicator@martin.zurowietz.de/extension.js
feb 18 08:39:42 jimmy-xps-9360 gnome-shell[1710]:  extracting: /home/jimmy/.local/share/gnome-shell/extensions/always-indicator@martin.zurowietz.de/metadata.json
mar 22 09:56:17 jimmy-xps-9360 gnome-shell[1650]:   inflating: /home/jimmy/.local/share/gnome-shell/extensions/always-indicator@martin.zurowietz.de/Settings.js
mar 22 09:56:17 jimmy-xps-9360 gnome-shell[1650]:   inflating: /home/jimmy/.local/share/gnome-shell/extensions/always-indicator@martin.zurowietz.de/extension.js
mar 22 09:56:17 jimmy-xps-9360 gnome-shell[1650]:   inflating: /home/jimmy/.local/share/gnome-shell/extensions/always-indicator@martin.zurowietz.de/prefs.js
mar 22 09:56:17 jimmy-xps-9360 gnome-shell[1650]:    creating: /home/jimmy/.local/share/gnome-shell/extensions/always-indicator@martin.zurowietz.de/schemas/
mar 22 09:56:17 jimmy-xps-9360 gnome-shell[1650]:   inflating: /home/jimmy/.local/share/gnome-shell/extensions/always-indicator@martin.zurowietz.de/schemas/org.gnome.shell.extensions.always-indicator.gschema.xml
mar 22 09:56:17 jimmy-xps-9360 gnome-shell[1650]:   inflating: /home/jimmy/.local/share/gnome-shell/extensions/always-indicator@martin.zurowietz.de/schemas/gschemas.compiled
mar 22 09:56:17 jimmy-xps-9360 gnome-shell[1650]:   inflating: /home/jimmy/.local/share/gnome-shell/extensions/always-indicator@martin.zurowietz.de/settings.ui
mar 22 09:56:17 jimmy-xps-9360 gnome-shell[1650]:  extracting: /home/jimmy/.local/share/gnome-shell/extensions/always-indicator@martin.zurowietz.de/metadata.json
mar 28 20:57:38 jimmy-xps-9360 gnome-shell[1685]:   inflating: /home/jimmy/.local/share/gnome-shell/extensions/always-indicator@martin.zurowietz.de/Settings.js
mar 28 20:57:38 jimmy-xps-9360 gnome-shell[1685]:   inflating: /home/jimmy/.local/share/gnome-shell/extensions/always-indicator@martin.zurowietz.de/extension.js
mar 28 20:57:38 jimmy-xps-9360 gnome-shell[1685]:   inflating: /home/jimmy/.local/share/gnome-shell/extensions/always-indicator@martin.zurowietz.de/prefs.js
mar 28 20:57:38 jimmy-xps-9360 gnome-shell[1685]:    creating: /home/jimmy/.local/share/gnome-shell/extensions/always-indicator@martin.zurowietz.de/schemas/
mar 28 20:57:38 jimmy-xps-9360 gnome-shell[1685]:   inflating: /home/jimmy/.local/share/gnome-shell/extensions/always-indicator@martin.zurowietz.de/schemas/org.gnome.shell.extensions.always-indicator.gschema.xml
mar 28 20:57:38 jimmy-xps-9360 gnome-shell[1685]:   inflating: /home/jimmy/.local/share/gnome-shell/extensions/always-indicator@martin.zurowietz.de/schemas/gschemas.compiled
mar 28 20:57:38 jimmy-xps-9360 gnome-shell[1685]:   inflating: /home/jimmy/.local/share/gnome-shell/extensions/always-indicator@martin.zurowietz.de/settings.ui
mar 28 20:57:38 jimmy-xps-9360 gnome-shell[1685]:  extracting: /home/jimmy/.local/share/gnome-shell/extensions/always-indicator@martin.zurowietz.de/metadata.json
```

I also tried

```
$ journalctl /usr/bin/gnome-shell | grep "indicator"
```
which returns many more lines due to other extensions I have that use the word "indicator" (like: clipboard-indicator, cpufreqindicator). I see nothing new in this output about your extension. :\

**Comment by mzur** [comment]
Maybe the other extensions interfere with this extension. Could you disable all extensions but this one,  then restart GNOME and try again?

**Comment by amivaleo** [comment]
> Maybe the other extensions interfere with this extension. Could you disable all extensions but this one, then restart GNOME and try again?

I did that.

> I disabled all the extensions I use except for this one, rebooted my laptop and then I gave

**Comment by mzur** [comment]
Which version of this extension do you use? The version number is located in `~/.local/share/gnome-shell/extensions/always-indicator@martin.zurowietz.de/metadata.json`.

**Comment by amivaleo** [comment]
Version: 2.

It's the latest available on the website.

**Comment by mzur** [comment]
That's correct, too. If there is no error in the log while all extensions but this one are disabled and it still doesn't work, I'm at a dead end here, sorry.

**Comment by amivaleo** [comment]
I don't really understand.

I've tried your extension on a fresh new virtual machine (gnome-boxes) in which I installed the very same distro: fedora 31.

There it works!

I tried >again< to disable all but this extension on my computer -> reboot -> it doesn't work! o.o

I see no errors whatsoever, but I'm not much skilled. I only checked the "journalctl" and in "looking glass". Everything looks ok there...

Maybe my system is missing some package...? Some "libnotify-123" or something...?

Do you if that could be the case? I mean, do you know if that dot depends on some particular package?

**Comment by mzur** [comment]
Does the regular popup notification appear with `notify-send`? If so, try to send 4-5 notifications in a row and see if the default notification dot is displayed.

**Comment by amivaleo** [comment]
No... It's been a while since the last time I saw that dot. 🤔

I compared the full list of installed packages on my computer with those installed on the virtual machine. I am pretty sure none of the very few missing packages are related to this issue.

It might be related to the changes I made to the stylesheet of the default gnome shell theme. Indeed, I recompiled the theme and the dot appeared. But, after rebooting, the issue is still there.

I need further checks, that I will in long times and on my own. In the meanwhile I close this issue and eventually come back here with a solution, if I find it. This seems to be an issue that only affects me, and the only "strongly-not-suggested" action I did was to recompile the standard theme. There is where I'll focus my attention to get this fixed.

Thank you!
